android/library/src/main/java/com/liferay/mobile/screens/imagegallery/interactor/ImageGalleryInteractorListener.java

```diff
@@ -8,13 +22,44 @@
  */
 public interface ImageGalleryInteractorListener extends BaseListInteractorListener<ImageEntry> {
 
+	/**
+	 * Called when an item in the list is deleted.
+	 *
+	 * @param imageEntryId
+	 */
 	void onImageEntryDeleted(long imageEntryId);
 
+	/**
+	 * Called when an item in the list was chosen and it's ready for upload.
+	 *
+	 * @param picturePath
+	 */
 	void onPicturePathReceived(String picturePath);
 
+	/**
+	 * Called when the item was successfully uploaded.
+	 *
+	 * @param entry
+	 */
 	void onPictureUploaded(ImageEntry entry);
 
+	/**
+	 * Retrieves constantly the progress until the picture is successfully uploaded.
+	 * This method retrieves the bytes sent and the total.
+	 *
+	 * @param totalBytes
+	 * @param totalBytesSent
+	 */
 	void onPictureUploadProgress(int totalBytes, int totalBytesSent);
 
+	/**
+	 * Retrieves constantly information until the picture is successfully uploaded.
+	 * This method retrieves the picture path, title, description and changelog.
+	 *
+	 * @param picturePath
+	 * @param title
+	 * @param description
+	 * @param changelog
+	 */
 	void onPictureUploadInformationReceived(String picturePath, String title, String description, String changelog);
```

Comment:
onPictureUploadInformationReceived is called only once when user fill the upload form